cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1487,6 +1487,215 @@ action.summary_index._name = cyences
 action.summary_index.category = O365
 action.summary_index.alert_name = O365 - Application Change/Update
 
+[O365 - Login failure due to multi factor authentication]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule =	29 * * * *
+description = This alert will show the login failure due to multi factor authentication. \
+\
+Data Collection - Office 365 management activity data (Splunk Add-on for Office 365). 
+dispatch.earliest_time = -85m@s
+dispatch.latest_time = -25m@s
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_o365` sourcetype="o365:management:activity" Workload=AzureActiveDirectory Operation=UserLoginFailed (LogonError="DeviceAuthenticationRequired" OR LogonError="UserStrongAuthClientAuthNRequiredInterrupt") \
+| eval ExtendedProperties=mvzip('ExtendedProperties{}.Name','ExtendedProperties{}.Value'," : ") \
+| stats count, max(_time) as Last_Failed_Login, values(LogonError) as LogonError, values(ExtendedProperties) as ExtendedProperties by user, ClientIP \
+| iplocation ClientIP \
+| fillnull Country, Region, City value="-" \
+| eval Location=ClientIP." (".count.") | ".Country." | ".Region." | ".City \
```

Comment:
Similar way let's also add count with LogonError.

Reply:
We will get same error value multiple times as we will have to add LogonError field into by fields of first stats comamnd that is why i haven't added count value. Ex.

| user | Error |
|-------| ------|
| xyz | UserStrongAuthClientAuthNRequiredInterrupt(5) <br/>UserStrongAuthClientAuthNRequiredInterrupt(8) <br/>UserStrongAuthClientAuthNRequiredInterrupt(4) |

Let me  know if we still want to add.

Reply:
That is correct. This is fine then. Thanks for clarifing.